src/routes.ts

```diff
@@ -72,3 +72,10 @@ export const walletBondsRoute: Route<void> = {
     return this.template
   }
 }
+export type WalletAssetDetailsRouteParams = { symbol: string }
+export const walletAssetDetailsRoute: Route<void> = {
+  template: `${walletBaseRoute.template}/asset-details/:symbol`,
+  path() {
```

Comment:
If you have a parameter `:symbol` in `template` defined, then path function might be something like this:
```
path(symbol: string) =>  `${walletBaseRoute.path}/asset-details/${symbol}`
```